src/Hystrix.Dotnet.AspNet/HystrixCommandGroupCollection.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Configuration;
+
+namespace Hystrix.Dotnet.AspNet
+{
+    public class HystrixCommandGroupCollection : ConfigurationElementCollection
+    {
+        protected override ConfigurationElement CreateNewElement()
+        {
+            return new HystrixCommandGroupElement();
+        }
+
+        protected override object GetElementKey(ConfigurationElement element)
+        {
+            var commandGroup = element as HystrixCommandGroupElement;
```

Comment:
Unless you really want to have an exception with the parameter name 'element', you can use the cast operator to `HystrixCommandGroupElement` here, then you can get rid of the verification below in which you throw the `InvalidCastException`. #reallyNotABigDeal